### DISCUSSION TITLE
react query context

### DISCUSSION THREAD

**Discussion by wulimao49** [wevm/wagmi#661] [Ideas]
now i have my own react query client defined and i don't want to override the react query client inside wagmi to match my need,
i think is is better to use a seperate react query context in wagmi?

https://tanstack.com/query/v4/docs/guides/migrating-to-react-query-4#custom-contexts-for-multiple-providers

**Comment by aj-may** [comment]
Made a PR for this:
https://github.com/wagmi-dev/wagmi/pull/963
